RCTAppleHealthKit/RCTAppleHealthKit+Queries.m

```diff
@@ -241,6 +241,57 @@ - (void)fetchSamplesOfType:(HKSampleType *)type
     [self.healthStore executeQuery:query];
 }
 
+- (void)fetchClinicalRecordsOfType:(HKClinicalType *)type
+                         predicate:(NSPredicate *)predicate
+                         ascending:(BOOL)asc
+                             limit:(NSUInteger)lim
```

Comment:
We could change `asc` and `lim` to `ascending` and `limit` here, this is how these guys are named in other places, ok?

Reply:
It looks like all cases except `fetchWorkoutForPredicate` in this file use the convention of `ascending:(BOOL)asc` and `limit:(NSUInteger)lim` as used here.

Reply:
No problems then 😄